mmv1/products/vmwareengine/Subnet.yaml

```diff
@@ -0,0 +1,168 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Subnet'
+base_url: '{{parent}}/subnets'
+create_url: '{{parent}}/subnets/{{name}}?update_mask=ip_cidr_range'
+self_link: '{{parent}}/subnets/{{name}}'
+update_mask: true
+create_verb: :PATCH
+update_verb: :PATCH
+skip_delete: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/vmware-engine/docs/reference/rest/v1/projects.locations.privateClouds.subnets'
+description: |
+  Subnet in a private cloud. Either `management` subnets (such as vMotion) that  are read-only,
```

Comment:
This resource is only for user defined subnets. Can you please modify the description? Thanks.

Reply:
Done